src/components/EpisodeCardsList/EpisodeCardsList.js

```diff
@@ -0,0 +1,34 @@
+import EpisodeCard from '../EpisodeCard/EpisodeCard'
+import LoadingSpinner from '../LoadingSpinner/LoadingSpinner'
+import PropTypes from 'prop-types'
+import styled from 'styled-components/macro'
+import useEpisodesByFeedId from '../../hooks/useEpisodesByFeedId'
+
+EpisodeCardsList.propTypes = {
+  id: PropTypes.number.isRequired,
+}
+
+export default function EpisodeCardsList({ id }) {
+  const { episodesByFeedId, isQuerying } = useEpisodesByFeedId(id)
+
+  if (isQuerying) {
+    return <LoadingSpinner />
+  }
+
+  return (
+    <EpisodeList>
+      {episodesByFeedId.items.map(episode => (
+        <li key={episode.id}>
+          <EpisodeCard episode={episode} />
+        </li>
+      ))}
+    </EpisodeList>
+  )
```

Comment:
```suggestion
  return (
  isQuerying ? (
  <LoadingSpinner />
  ) : (
    <EpisodeList>
      {episodesByFeedId.items.map(episode => (
        <li key={episode.id}>
          <EpisodeCard episode={episode} />
        </li>
      ))}
    </EpisodeList>
    )
  )
```

This is how I would structure the code, but yours is working well as well. Just for ideas 😉

Reply:
I had it like yours before, but then leon showed me the approach im using now and i like it, because it separated the code and i find it easier to read.